src/Azure/Azure.Quantum.Client/generated/IProvidersOperations.cs

```diff
@@ -0,0 +1,68 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Quantum.Client
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Models;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// ProvidersOperations operations.
+    /// </summary>
+    public partial interface IProvidersOperations
```

Comment:
should this be IProviderOperations instead? we might need to update the swagger for this.
Andres's comment also makes sense we can use IProviderStatusOperations.Get

Reply:
I think the plural `Providers` does make some sense because it's operating on the list of providers, rather than a single provider. But I don't have a strong opinion.